src/session.rs

```diff
@@ -387,13 +387,15 @@ impl Session {
                 stream_mid_rid?.ssrc()
             } else {
                 if let Some(stream) = stream_mid_rid {
-                    if stream.ssrc() != ssrc {
-                        // We got a change in main SSRC for this stream.
-                        let ssrc_from = stream.ssrc();
-                        self.change_stream_rx_ssrc(ssrc_from, ssrc);
+                    let existing_ssrc = stream.ssrc();
+                    if existing_ssrc != ssrc {
+                        info!("Ignoring remote attempt to remap SSRC for MID: {mid} and RID: {rid}: {existing_ssrc} -> {ssrc}")
                     }
+
+                    existing_ssrc
```

Comment:
This doesn't quite work because of this code in `Session::handle_rtp`

https://github.com/algesten/str0m/blob/53d1140e74abfee114d485f6aff04721229ce45c/src/session.rs#L470

we are going to decide that this is a repair stream or not. I think we ought to change the logic in `Session::handle_rtp` to rely on Pt to identify repair packets instead of the existing check

Reply:
Addressed in `cea2055`by relying on payload type instead of ssrc